apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx

```diff
@@ -0,0 +1,138 @@
+import { Text } from '_app/shared/text';
+import Alert from '_components/alert';
+import Icon, { SuiIcons } from '_components/icon';
+import LoadingIndicator from '_components/loading/LoadingIndicator';
+import { useAppSelector } from '_hooks';
+
+export function ValidatorsCard() {
+    const accountAddress = useAppSelector(({ account }) => account.address);
+    const {
+        data: stakeValidators,
+        isLoading,
+        isError,
+        error,
+    } = useGetDelegatedStake(accountAddress || '');
+
+    const totalActivePendingStake = useMemo(() => {
+        if (!stakeValidators) return 0n;
+        return stakeValidators.reduce(
+            (acc, { staked_sui }) => acc + BigInt(staked_sui.principal.value),
+            0n
+        );
+    }, [stakeValidators]);
+
+    const stakingEnabled = useFeature(FEATURES.STAKING_ENABLED).on;
+
+    if (isLoading) {
+        return (
+            <div className="p-2 w-full flex justify-center items-center h-full">
+                <LoadingIndicator />
+            </div>
+        );
+    }
+
+    if (isError) {
+        return (
+            <div className="p-2 w-full flex justify-center items-center h-full">
+                <Alert className="mb-2">
+                    <strong>{error?.message}</strong>
+                </Alert>
+            </div>
+        );
+    }
+
+    return (
+        <div className="flex flex-col flex-nowrap h-full overflow-x-scroll w-full">
+            <BottomMenuLayout>
+                <Content>
+                    <div className="mb-4">
+                        <Card
+                            padding="none"
+                            header={
+                                <div className="py-2.5">
+                                    <Text
+                                        variant="captionSmall"
+                                        weight="semibold"
+                                        color="steel-darker"
+                                    >
+                                        Staking on {stakeValidators.length}
+                                        {stakeValidators.length > 1
+                                            ? ' VALIDATORS'
+                                            : ' VALIDATOR'}
+                                    </Text>
+                                </div>
+                            }
+                        >
+                            <div className="flex divide-x divide-solid divide-gray-45 divide-y-0">
+                                <CardItem title="Your Stake">
+                                    <StakeAmount
+                                        balance={totalActivePendingStake}
+                                        variant="heading4"
+                                    />
+                                </CardItem>
+                                {/* TODO: show the actual Rewards Collected value https://github.com/MystenLabs/sui/issues/3605 */}
```

Comment:
Is this actually still relevant? I thought we could actually compute the rewards now?